team/_posts/2024-04-29-john-sj-anderson.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: member
+title: John SJ Anderson
+position: Software engineer
+handle: genehack
+github: genehack
+fediverse:
+  server: dementedandsadbut.social
+  handle: "@genehack"
+image: /images/team/john-sj-anderson.jpg
+website: https://genehack.org
+---
+
+I joined the Bedford Lab near the end of 2024. I have a PhD in
```

Comment:
> near the end of 2024

Hmm, you're a bit ahead of schedule 😄

Reply:
🤦 good catch — updated with "end of April, 2024"